app/routes/_gcn.docs.contributing.project/route.mdx

```diff
@@ -0,0 +1,196 @@
+---
+handle:
+  breadcrumb: Project Board
+---
+
+import { Table } from '@trussworks/react-uswds'
+
+import issueScreenshot from './issue.png'
+import openTicketScreenshot from './openTicket.png'
+import projectBoardScreenshot from './projectBoard.png'
+import ticketMetadata from './ticketMetadata.png'
+
+# Github Project Board
+
+[GCN Project Board](https://github.com/orgs/nasa-gcn/projects/6)
+
+## Philosophy
+
+We use github Projects to organize our development process.
+We chose github Projects becase we already extensively use github Issues and Milestones.
+[Github's Projects Documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects)
+
+We have a total of six possible ticket states.
+
+<Table bordered compact stackedStyle="headers">
+  <thead>
+    <tr>
+      <th scope="col">Status</th>
+      <th scope="col">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>No Status</td>
+      <td>
+        These are tickets that have not been fully groomed or pointed yet.
```

Comment:
Can you think of a work other than groomed? It sounds creepy.

Reply:
I agree that it's creepy, but it's also standard. I think the second most common word after backlog grooming (or ticket grooming) is refinement. does fully refined sound better?

Reply:
I think so.